### PR TITLE
doc: Explain payload formatter types

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add introduction to types of Payload Formatters, from feedback that it wasn't clear how to create custom payload formatters.

Also consolidated CLI and Console payload formatter docs

#### Screenshots

Screenshot 2021-01-05 at 12 15 36
Screenshot 2021-01-05 at 12 33 51

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by johanstokking — APPROVED** [review]

**Review by adriansmares — APPROVED** [review]

File: doc/content/integrations/payload-formatters/_index.md [hunk 1]
```diff
@@ -5,10 +5,28 @@ description: ""
...
 
 <!--more-->
 
+## Types of Payload Formatters
+
+{{% tts %}} supports three types of Payload Formatters: Javascript, CayenneLPP, and Device Repo payload formatters.
+
+### Javascript
+
+{{% tts %}} allows you to write your own custom Payload Formatters in Javascript. To find out how to write your own custom payload formatters, see the [Javascript Payload Formatters section]({{< relref "javascript" >}}).
```
Comment (nejraselimovic) [inline doc/content/integrations/payload-formatters/_index.md:18]:
```suggestion
To find out how to write a custom Javascript payload formatter, see the [Javascript Payload Formatters]({{< relref "javascript" >}}) section.
```

File: doc/content/integrations/payload-formatters/_index.md [hunk 2]
```diff
@@ -5,10 +5,28 @@ description: ""
...
+
+### CayeneLPP
+
+{{% tts %}} can automatically decode CayenneLPP formatted payloads, no custom code neccessary. To find out how to use CayenneLPP Payload Formatters, see the [CayenneLPP section]({{< relref "cayenne" >}}).
```
Comment (nejraselimovic) [inline doc/content/integrations/payload-formatters/_index.md:22]:
```suggestion
{{% tts %}} can automatically decode CayenneLPP formatted payloads, no custom code neccessary. To find out how to use CayenneLPP payload formatters, see the [CayenneLPP]({{< relref "cayenne" >}}) section.
```

File: doc/content/integrations/payload-formatters/_index.md [hunk 3]
```diff
@@ -5,10 +5,28 @@ description: ""
...
+
+### Device Repository
+
+Device manufacturers may submit payload formatters designed to work with their devices. These are publicly available in {{% tts %}} [Device Repository](https://github.com/TheThingsNetwork/lorawan-devices/tree/master).
```
Comment (nejraselimovic) [inline doc/content/integrations/payload-formatters/_index.md:26]: Where exactly?

**Review by nejraselimovic — COMMENTED** [review]
I think we should replace **Device Repository** titles since the payload formatter type is called **Repository**. 

Also, I'm a bit disappointed when I click on the **Device Repository** section and see the same info as here (**Device Repository** in the overview page) - I'm expecting to have a more detailed info in the separate section. Maybe adding an example?

File: doc/content/integrations/payload-formatters/_index.md (hunk 3 above)
Comment (bafonins) [reply doc/content/integrations/payload-formatters/_index.md:26]: Would be nice to link to a guide on how to contribute payload formatters.

**Review by bafonins — APPROVED** [review]

File: doc/content/integrations/payload-formatters/_index.md (hunk 3 above)
Comment (benolayinka) [reply doc/content/integrations/payload-formatters/_index.md:26]: This is coming with https://github.com/TheThingsNetwork/lorawan-stack/issues/3522

**Pull request merged**